### ISSUE TITLE
Remove the original timers that were deprecated a while back

### ISSUE THREAD

**Issue by jeking3** [boostorg/timer#7]
Looking at http://www.boost.org/doc/libs/1_65_1/libs/timer/doc/index.html, it looks like there could be some code simplification by removing the original timers.

**Comment by pdimov** [comment]
Good idea in principle. I've marked these headers as deprecated in https://github.com/boostorg/timer/commit/075d6b25ab185526b0704fc9d1c982931729c35c. Hopefully they would become removable in a few releases (there are still too many uses of them in tests, and one in Boost.Test.)
